nsqd/nsqd.go

```diff
@@ -372,6 +372,7 @@ func (n *NSQD) LoadMetadata() error {
 				channel.Pause()
 			}
 		}
+		topic.UnPause()
```

Comment:
Above, you'll see that paused state is stored in the topic metadata, so it does:

```go
 if t.Paused {
 	topic.Pause()
}
```

So you can remove that now, but here only un-pause if the topic was not originally paused.

Reply:
Thanks, i will correct it